src/components/Dropdown.astro

```diff
@@ -0,0 +1,67 @@
+---
+import Icon from "astro-icon";
+import { pathWithoutLanguage } from "../routing/lang";
+import { classNames } from "../utils/classNames";
+
+type Props = {
+  item: string;
+  options: readonly string[];
+};
+
+const pathWithoutLang = pathWithoutLanguage(Astro.url.pathname);
+const { item, options } = Astro.props;
+---
+
+<div class="min-w-[6rem]">
+  <div class="relative mt-1">
+    <button
+      type="button"
+      class="peer relative w-full cursor-default rounded-md py-2 pl-3 pr-10 text-left text-white hover:bg-wsg-orange-400 hover:bg-opacity-50 focus:ring-1 focus:ring-wsi-blue-300 sm:text-sm"
+      aria-haspopup="listbox"
+      aria-expanded="true"
+      aria-labelledby="listbox-label"
+    >
+      <span class="block truncate text-base font-medium">{item}</span>
+    </button>
+    <ul
+      class="absolute z-10 mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm opacity-0 pointer-events-none hover:pointer-events-auto peer-focus:pointer-events-auto peer-focus:opacity-100 transition-opacity"
+      tabindex="-1"
+      role="listbox"
+      aria-labelledby="listbox-label"
+      aria-activedescendant="listbox-option-3"
+    >
+      {
+        options.map((option) => {
+          const isSelected = option === item;
+
+          return (
+            <li
+              onclick={
+                isSelected
+                  ? undefined
+                  : `location.href = '/${option.toLowerCase()}${pathWithoutLang}'`
+              }

```

Comment:
This is a very specific action for a component called `Dropdown` we should either allow specifying the `onclick` or rename the component to `LanguageDropdown`

Reply:
Oh, I forgot...